browser/gotrue.js

```diff
@@ -1 +1 @@
headers||{};var aud=options.audience||this.audience;if(aud){options.headers["X-JWT-AUD"]=aud}return this.jwt().then(function(token){return _this3.api.request(path,_extends({headers:Object.assign(options.headers,{Authorization:"Bearer "+token})},options)).catch(function(err){if(err instanceof _microApiClient.JSONHTTPError&&err.json){if(err.json.msg){err.message=err.json.msg}else if(err.json.error){err.message=err.json.error+": "+err.json.error_description}}return Promise.reject(err)})})}},{key:"getUserData",value:function getUserData(){return this._request("/user").then(this._saveUserData.bind(this)).then(this._refreshSavedSession.bind(this))}},{key:"_saveUserData",value:function _saveUserData(attributes,fromStorage){for(var key in attributes){if(key in User.prototype||key in forbiddenUpdateAttributes){continue}this[key]=attributes[key]}if(fromStorage){this._fromStorage=true}return this}},{key:"_processTokenResponse",value:function _processTokenResponse(tokenResponse){this.token=tokenResponse;var claims=void 0;try{claims=JSON.parse(urlBase64Decode(tokenResponse.access_token.split(".")[1]));this.token.expires_at=claims.exp*1e3}catch(e){console.error(new Error("Gotrue-js: Failed to parse tokenResponse claims: "+JSON.stringify(tokenResponse)))}}},{key:"_refreshSavedSession",value:function _refreshSavedSession(){if(isBrowser()&&localStorage.getItem(storageKey)){this._saveSession()}return this}},{key:"_saveSession",value:function _saveSession(){isBrowser()&&localStorage.setItem(storageKey,JSON.stringify(this._details));return this}},{key:"tokenDetails",value:function tokenDetails(){return this.token}},{key:"clearSession",value:function clearSession(){User.removeSavedSession();this.token=null;currentUser=null}},{key:"admin",get:function get(){return new _admin2.default(this)}},{key:"_details",get:function get(){var userCopy={};for(var key in this){if(key in User.prototype||key in forbiddenSaveAttributes){continue}userCopy[key]=this[key]}return userCopy}}],[{key:"removeSavedSession",value:function removeSavedSession(){isBrowser()&&localStorage.removeItem(storageKey)}},{key:"recoverSession",value:function recoverSession(apiInstance){if(currentUser){return currentUser}var json=isBrowser()&&localStorage.getItem(storageKey);if(json){try{var data=JSON.parse(json);var url=data.url,token=data.token,audience=data.audience;if(!url||!token){return null}var api=apiInstance||new _microApiClient2.default(url,{});return new User(api,token,audience)._saveUserData(data,true)}catch(ex){console.error(new Error("Gotrue-js: Error recovering session: "+ex));return null}}return null}}]);return User}();exports.default=User;function urlBase64Decode(str){var output=str.replace(/-/g,"+").replace(/_/g,"/");switch(output.length%4){case 0:break;case 2:output+="==";break;case 3:output+="=";break;default:throw"Illegal base64url string!"}var result=window.atob(output);try{return decodeURIComponent(escape(result))}catch(err){return result}}},{"./admin":3,"micro-api-client":1}]},{},[4]);
```

Comment:
hmm.. were you forced to modify this file? surprised to see that we checked in this build output here. but i dont know this library that well.

Reply:
No I was not forced, but I saw it was commited, so I compiled and pushed it too 👍